### ISSUE TITLE
2h weapon damage critical bug ! 

### ISSUE THREAD

**Issue by amounabik** [SunwellTracker/issues#2611]
Decription: 
I just bought an **arcanite reaper heirloom** from the sunwell store , but the minimum damage sometimes **is reduced heavily** without any reason , my damage is like 170-240 max , with the bug sometimes it turns out to 70-240 , i could fix this by equiping again the axe , but the bug occur many times to point , its not hot fix solution for me to just equip and and desequip again the heirloom axe ! hope it get fixed very soon ! 

How it works: 
the minimum damage is reduced heavily sometimes without having any curse or debuff actif from enemy 

How it should work: 
min max damage should scale with buffs and gears and enemy debuffs and curses properly 

Source (you should point out proofs of your report, please give us some source): 
![yeah](https://user-images.githubusercontent.com/58403708/69921161-18138500-1490-11ea-8f1c-0f794124437d.png)
![yeah2](https://user-images.githubusercontent.com/58403708/69921162-18138500-1490-11ea-8e0f-2f6068c72bf9.png)

**Comment by amounabik** [comment]
By the way its not visual bug , my damage as well got lowered so much i even hitting 70-100 dmg with overpower non crit instead 150-200 normally !

**Comment by Rosenkrantz** [comment]
I'm going to guess that you got disarmed before that.

I noticed this on Angrathar too a long time ago - after a disarm, your minimum damage is scaled down as if you wielded lv1 version of the weapon. You can fix this by re-equipping the weapon, but it's a bug nonetheless.

**Comment by amounabik** [comment]
i got disarmed the whole dung in blackfathom deeps , but still its a serious bug , when the minimum damage become very bad to point you dont do anymore damage

**Comment by Riztazz** [comment]
Only applies to heirloom weapons luckily, gonna fix it today

**Comment by amounabik** [comment]
Yeah , i can confirm that this happens when im disarmed im throught many sm dungeon runs and got many time disarmed as well by dinosaurs in stv , and the min damage reduced by disarm , hope it get fixed fast , and thanks

**Comment by AbraKabastard** [comment]
This was second of December. Is this resolved?
